packages/client/src/components/Location/LocationFinder.test.js

```diff
@@ -118,6 +118,75 @@ describe("LocationFinder", () => {
     );
     await waitFor(() => screen.getByTestId(LOCATION_FOUND));
   });
+  it("should render multiple housenumber types", async () => {
+    render(<Wrapper />, {}, locationFinderGraphQLMocks);
+
+    const inputPostalCode = screen.getByLabelText(
+      text.translation.common["postalcode label"]
+    );
+    const inputHouseNumber = screen.getByLabelText(
+      text.translation.common["housenumber label"]
+    );
+
+    expect(inputPostalCode).toHaveValue("");
+    expect(inputHouseNumber).toHaveValue("");
+
+    await act(async () => {
+      fireEvent.change(inputPostalCode, {
+        target: { value: "1024BV" },
+      });
+      fireEvent.change(inputHouseNumber, {
+        target: { value: "546 K 1" },
+      });
+    });
+
+    await waitFor(() =>
+      screen.queryByText(text.translation.common["address loading"])
+    );
+    await waitFor(() => screen.getByTestId(LOCATION_FOUND));
+
+    await act(async () => {
+      fireEvent.change(inputPostalCode, {
+        target: { value: "1024BV" },
+      });
+      fireEvent.change(inputHouseNumber, {
+        target: { value: "546k1" },
+      });
+    });
+
+    await waitFor(() =>
+      screen.queryByText(text.translation.common["address loading"])
+    );
+    await waitFor(() => screen.getByTestId(LOCATION_FOUND));
+
+    await act(async () => {
+      fireEvent.change(inputPostalCode, {
+        target: { value: "1024BV" },
+      });
+      fireEvent.change(inputHouseNumber, {
+        target: { value: "546 k1" },
+      });
+    });
+
+    await waitFor(() =>
+      screen.queryByText(text.translation.common["address loading"])
+    );
+    await waitFor(() => screen.getByTestId(LOCATION_FOUND));
+
+    await act(async () => {
+      fireEvent.change(inputPostalCode, {
+        target: { value: "1027AE" },
+      });
+      fireEvent.change(inputHouseNumber, {
+        target: { value: "20 H L" },
+      });
+    });
```

Comment:
Awesome tests dude! You got it working (-:

Maybe you can add come more, like:
- 1024BV, 546k 1
- 1024BV, 546k 1
- 1024BV, 54 6k 1 (not found?)

- 1027AE, 20hl
- 1027AE, 20HL
- 1027A, 20 h L
- 1027A, " 20 hl "

And maybe, if you want to be even more awesome, you can even add `matches` to the mocks and detect for each input if the AutoSuggest is open with:
- "546 K 1",
- "546 K 10",